example/src/main/java/com/stripe/example/activity/PaymentActivity.java

```diff
@@ -1,338 +1,46 @@
 package com.stripe.example.activity;
 
-import android.content.BroadcastReceiver;
-import android.content.Context;
-import android.content.Intent;
-import android.content.IntentFilter;
 import android.os.Bundle;
-import android.support.annotation.NonNull;
-import android.support.v4.app.DialogFragment;
-import android.support.v4.content.LocalBroadcastManager;
 import android.support.v7.app.AppCompatActivity;
-import android.view.View;
 import android.widget.Button;
 import android.widget.EditText;
 import android.widget.ListView;
-import android.widget.SimpleAdapter;
 import android.widget.Spinner;
 
-import com.jakewharton.rxbinding.view.RxView;
-import com.stripe.android.Stripe;
-import com.stripe.android.TokenCallback;
-import com.stripe.android.model.Card;
-import com.stripe.android.model.Token;
 import com.stripe.example.R;
-import com.stripe.example.dialog.ErrorDialogFragment;
-import com.stripe.example.dialog.ProgressDialogFragment;
-import com.stripe.example.service.TokenIntentService;
-
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.concurrent.Callable;
-
-import rx.Observable;
-import rx.android.schedulers.AndroidSchedulers;
-import rx.functions.Action0;
-import rx.functions.Action1;
-import rx.schedulers.Schedulers;
-import rx.subscriptions.CompositeSubscription;
+import com.stripe.example.module.DependencyHandler;
 
 public class PaymentActivity extends AppCompatActivity {
 
-    /*
-     * Change this to your publishable key.
-     *
-     * You can get your key here: https://manage.stripe.com/account/apikeys
-     */
-    private static final String PUBLISHABLE_KEY = "pk_test_6pRNASCoBOKtIshFeQd4XMUh";
-    private static final String CURRENCY_UNSPECIFIED = "Unspecified";
-
-    private ProgressDialogFragment progressFragment;
-
-    // Controls for card entry
-    private EditText cardNumberEditText;
-    private EditText cvcEditText;
-    private Spinner monthSpinner;
-    private Spinner yearSpinner;
-    private Spinner currencySpinner;
-
-    // Fields used to display the returned card tokens
-    private ListView listView;
-    private SimpleAdapter simpleAdapter;
-    private List<Map<String, String>> cardTokens = new ArrayList<Map<String, String>>();
-
-    // A CompositeSubscription object to make cleaning up our rx subscriptions simpler
-    private CompositeSubscription compositeSubscription;
-
-    // A receiver used to listen for local broadcasts.
-    private TokenBroadcastReceiver tokenBroadcastReceiver;
+    private DependencyHandler mDependencyHandler;
 
     @Override
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.payment_activity);
 
-        progressFragment = ProgressDialogFragment.newInstance(R.string.progressMessage);
+        mDependencyHandler = new DependencyHandler(
```

Comment:
oh, no. it wasn't using the hungarian notation at all. i swapped it out because i almost always see that in android projects.